.github/workflows/linters.yml

```diff
@@ -45,4 +45,18 @@ jobs:
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
       - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
\ No newline at end of file
+        run: npx stylelint "**/*.{css,scss}"
```

Comment:
- Kindly enable your Linters in your pull request. If you need the setup files you can watch [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).